Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM python:3.7-slim-stretch
-MAINTAINER Transferwise
+FROM python:3.12-slim
+MAINTAINER Wise
```

Comment:
`MAINTAINER` is a deprecated field, https://docs.docker.com/reference/dockerfile/#maintainer-deprecated 

`LABEL "MAINTAINER"="Wise"` could be used